packages/manager/apps/pci-rancher/src/pages/dashboard/_layout.tsx

```diff
@@ -37,7 +38,10 @@ export default function DashboardPage() {
   if (error) {
     return (
       <Suspense>
-        <ErrorBanner error={error} />
+        <ErrorBanner
+          onRedirectHome={() => navigate('')}
+          error={error?.response}
```

Comment:
Since you're testing that error is truthy, there is no need to use optional chaining
```suggestion
          error={error.response}
```